docs/setup/schema.md

```diff
@@ -79,7 +79,16 @@ The `InformaticsGateway` configuration section contains the following sub-sectio
       }
     },
     "storage": {
-      "temporary": "/payloads"
+      "temporary": "./payloads",
+      "bucketName": "monaideploy",
+      "settings": {
+        "endpoint": "localhost:9000",
+        "accessKey": "username",
```

Comment:
I'd assume credentials here are are for local testing, and they are tokens etc will be retrieved/provided in a secure way on the deployed instance.

Reply:
This is part of the documentation, as an example.  
The config file can be stored as a CONFIGMAP in k8s and provided to the container.